skopt/utils.py

```diff
@@ -15,12 +15,14 @@
 from .learning.gaussian_process.kernels import HammingKernel
 from .learning.gaussian_process.kernels import Matern
 
+from .space import check_dimension
+from .space import Categorical
+
 __all__ = (
     "load",
     "dump",
 )
 
-
```

Comment:
Please don't delete these blank lines. We follow https://www.python.org/dev/peps/pep-0008/ for code formatting (`pip install pep8` and then `pep8 <filename>` to get advice about formatting). There should be two blank lines between functions, but only one blank line between methods of a class.